pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2818,6 +2819,18 @@ func handlePhysicalIOAdapterListImpl(ctxArg interface{}, key string,
 					}
 					aa.AddOrUpdateIoBundle(log, vfIb)
 				}
+			} else if ib.Type == types.IoVCAN {
+				// Initialize (create and enable) Virtual CAN device
+				err := setupVCAN(ib)
+				if err != nil {
+					log.Errorf("setupVCAN failed")
```

Comment:
the error is already logged and handled in setupVCAN() and setupCAN() functions. I don't want to return at this point because they are not fatal, so I think it's safe to not print the message "setupVCAN failed". I agree, it's useless since the error description will be already informed at this point...